app/Api/Procedure/TaskProcedure.php

```diff
@@ -166,6 +167,8 @@ public function updateTask($id, $title = null, $color_id = null, $owner_id = nul
             'priority' => $priority,
             'tags' => $tags,
             'date_started' => $date_started,
+			'time_estimated' => $time_estimated,
```

Comment:
This is a minor detail, but can you keep the code indentation consistent with the rest.

Thanks